### DISCUSSION TITLE
Request codes

### DISCUSSION THREAD

**Discussion by tzhu-bio** [GreenleafLab/ArchR#1073] [General]
![image](https://user-images.githubusercontent.com/54537420/136921353-45bdddef-9b93-4ac0-9420-30c2c50542fb.png)

Is it convenient to provide the code for this figure? Thanks!

**Comment by rcorces** [comment]
the individual heatmaps were made using the `ArchR:::.ArchRHeatmap()` function. The rest of the figure was compiled in Illustrator
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/ArchRHeatmap.R#L4
